WordPress/src/main/java/org/wordpress/android/util/image/ImageManager.kt

```diff
@@ -33,6 +36,25 @@ class ImageManager @Inject constructor(val placeholderManager: ImagePlaceholderM
         fun onResourceReady(resource: T)
     }
 
+    /**
+     * Return true if this [Context] is available.
+     * Availability is defined as the following:
+     * + [Context] is not null
+     * + [Context] is not destroyed (tested with [FragmentActivity.isDestroyed] or [Activity.isDestroyed])
+     */
+    private fun Context?.isAvailable(): Boolean {
+        if (this == null) {
```

Comment:
is this check necessary? if `this == null` then this code shouldn’t be running IIUC

Reply:
It's not required for this particular crash, #8591, but this new method is going to be used on almost all Glide calls, so that check does't hurt.

Reply:
I meant - does that expression make sense?

Reply:
It should be ok, we've defined an extension on a nullable receiver type, and can check for null inside the body. https://kotlinlang.org/docs/reference/extensions.html#nullable-receiver

Btw, i'm not expert in Kotlin.